crates/dojo/bindgen/src/plugins/typescript/generator/function.rs

```diff
@@ -12,7 +12,7 @@ impl TsFunctionGenerator {
     fn check_imports(&self, buffer: &mut Buffer) {
         if !buffer.has("import { DojoProvider } from ") {
             buffer.insert(0, "import { DojoProvider } from \"@dojoengine/core\";".to_owned());
-            buffer.insert(1, "import { Account } from \"starknet\";".to_owned());
+            buffer.insert(1, "import { Account, BigNumberish } from \"starknet\";".to_owned());
```

Comment:
```suggestion
            buffer.insert(1, "import { Account, JS_BIGNUMBERISH } from \"starknet\";".to_owned());
```
Use the newly defined constant.